src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1698,9 +1668,6 @@
     short-summary: Create a virtual network.
     long-summary: You may also create a subnet at the same time by specifying a subnet name and (optionally) an address prefix.
     examples:
-        - name: Create a basic virtual network.
```

Comment:
I would argue this should stay, but it should call out that it creates a virtual network with no subnets.

Reply:
I think the fact that the `--address-prefix` argument exists and is documented as both being optional and creating a subnet should be enough. Is there evidence that users are confused by creating VNETs with or without subnets? Is there any reason to create a VNET without one, on Azure? Does it just force a default `255.255.255.0` netmask? This seems more like a general documentation issue than example problem.

Reply:
I just know the default behavior of `vnet create` is to create a VNet with no subnets at all. Someone might assume that it would create a default subnet or something like that but it doesn't.